src/base/scheduler/day.h

```diff
@@ -0,0 +1,17 @@
+#include <QList>
+#include <QObject>
+
+namespace Scheduler
+{
+    class TimeRange;
+
+    class Day : public QObject
```

Comment:
A tip: there is no need to inherit QObject here.

Reply:
Thanks! 👍